packages/story-editor/src/components/publishModal/mainContent/mandatoryStoryInfo.js

```diff
@@ -42,15 +45,55 @@ const _TextArea = styled(TextArea)`
   margin: 8px 0;
 `;
 
-const MandatoryStoryInfo = ({
-  handleUpdateStoryInfo,
-  inputValues: _inputValues,
-}) => {
-  const [inputValues, setInputValues] = useState(_inputValues);
+const MandatoryStoryInfo = () => {
```

Comment:
💭 Should this be called something different now? Like `MainStoryInfo`?

Reply:
yes absolutely. I'll do it separate though 🙃